portfolio/src/main/webapp/information.html

```diff
@@ -0,0 +1,132 @@
+    •	Data types, console I/O, control structures, and libraries<br>
+    </div> 
+    </p>
+  <div id="project2" style="display:none">       
+   <p>
+        House Renovation – Python: Determined the number of boxes needed for my parent’s house renovation<br>
+        •	Calculated intelligently via final cost several other factors <br>
+            o	Area of the room, number of tiles per box, dimensions of each tile, and different brands
+    </p>
+    </div> 
+    <div id="project3" style="display:none">       
+    <p>
+    Automated Blinds – C++ | Arduino: Voice-controlled automated blinds
+    •	IoT project: Used Adafruit to create an open/close switch, with IFTTT platform 
+        implemented the trigger that changes the switch based on voice commands through Google Assistant 
+
+    </p>
+    </div> 
+</div> 
+
+<div class="experience">
+    <h1 style="text-align:center;font-size:30px;">Experience</h1>   
+   <p>  <a onclick="experience1()" id="experience"><img src="/images/sitel.jpg" class="picture1" /></a> 
+        <a onclick="experience2()"id="experience"><img src="/images/santaFe.jpg" class="picture2"/></a>
+        <a onclick="experience3()" style="font-size: 350px;background-position: right top" id="experience"> +</a> 
+    </p>
+
+    <div id="experience1" style="display:none">       
+    <p>
+    Sitel Group | Bogota, Colombia [13 weeks]						   	     2019
+	Bilingual call center agent as a representative of Just Energy Group Inc.
+        •Evaluated customer’s info to explore solutions while maintaining high-quality service
+        •Inbound calls sales team: Promoted products and services to customers during calls
+
+    </p>
+    </div> 
+    <div id="experience2" style="display:none">       
+    <p>
+    Santa Fe College | Gainesville, FL 				      	   Summer 2019 – Summer 2020
+    Math Tutor
+        •College Reading & Learning Association (CRLA) Level 1 certified tutor
+        •Collaborated with students from various math courses
+        •Assisted students to create studying strategies while supporting them with classwork
+    </p>
+    </div> 
+        <div id="experience3" style="display:none">       
+    <p>
+    Help me gain experience! 
+    </p>
+    </div>        
+
+</div>
+
+<div class="about">
+    <h1 style="text-align:center;font-size:30px;">About</h1>
+    <h2 style="text-align:left center">Thomas Pena </h2>     
+    <a>
+    <p style="right:25px; position: absolute;font-size:30px">
+    [super cool description goes here]
+    University of Florida UF College of Engineering in Gainesville, FL Graduation: May 2023<br>
+    Bachelor of Science: Computer Science<br>
+    Society of Hispanic Professional Engineers (SHPE UF)<br>
+    Triathlon Club at the University of Florida</p> 
+    <img src="/images/Pena, Thomas.jpg" class="profile"/>    
+    </a>
+</div>
+
+<div class="blog">
+    <h1 style="text-align:center;font-size:30px;color:whitesmoke">Blog</h1>  
+     <textarea>
```

Comment:
The closing tag for textarea looks a bit off in terms of formatting. Try copy + pasting your code into https://prettier.io/playground/ and see if it will format it a little better.